prplmesh/Makefile

```diff
@@ -21,6 +21,9 @@ PKG_LICENSE_FILES:=LICENSE
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk
 
+CMAKE_OPTIONS += -D_FLAVOUR_OPENWRT_=TRUE
+MAKE_VARS += PLATFORM=linux FLAVOUR=openwrt
```

Comment:
I don't think these variables are used. At least, I can build perfectly without it.